meadow/contracts/BasicContract.hs

```diff
@@ -0,0 +1,73 @@
+module BasicContract where
+
+import           Marlowe
+
+{-# ANN module "HLint: ignore" #-}
```

Comment:
Is this necessary? It seems it has no effect

Reply:
ah this one is because the contract files tend not to follow hlint rules, many of them are not really suitable for the users we will have e.g. eta-reduce